web/hand_tool.js

```diff
@@ -39,9 +39,9 @@ var HandTool = (function HandToolClosure() {
 
     this.handTool = new GrabToPan({
       element: this.container,
-      onActiveChanged: function(isActive) {
-        this.eventBus.dispatch('handtoolchanged', {isActive: isActive});
-      }.bind(this)
+      onActiveChanged(isActive) {
```

Comment:
It looks like this change breaks the hand tool menu item in the secondary toolbar. Toggling the hand tool does work, but the menu no longer disappears after toggling and the text is not updated anymore. I'm also not sure if `this` is bound correctly without `bind(this)` in this case.